### PR TITLE
fix: issue #77

### LINKED ISSUE #77: [feat][add link to the logo]
### Would you like to have a new feature? Please describe.

add a hyperlink to the logo, to send the user to the home page;

### How would this feature be beneficial for the project

- Yes, it will improve the SEO of the website.

- A user often clicked on a site logo to get to your site, so it is consistent and makes perfect sense for any click on the same logo to take you back to that same entry point.

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by ydv-ankit**
## Fixes Issue

This PR fixes the following issues:
add link to navbar logo

closes #77 

## Changes proposed

Here comes all the changes proposed through this PR

## Check List (Check all the boxes which are applicable) 

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

same, just linked to homepage

Add all the screenshots which support your changes

### CODE REVIEW

**Review by Vivekgaddigi — APPROVED** [review]
Looks good to me

File: client/src/components/Navbar/index.jsx [hunk 1]
```diff
@@ -6,7 +6,9 @@ const Navbar = () => {
   return (
     <nav className="primary-nav">
       <div>
-        <img src={Logo} alt="Roadmap Runner logo" />
+        <a href="/">
+          <img src={Logo} alt="Roadmap Runner logo" />
+        </a>
```
Comment (siddhigate) [inline client/src/components/Navbar/index.jsx:11]: Can you use the `Link` component of the `react-router-dom` instead of the anchor tag?

**Review by Shevilll — APPROVED** [review]

**Comment by AnuragThePathak** [comment]
@ydv-ankit are you still working on it?

**Comment by ydv-ankit** [comment]
check for PR: #88

**Pull request closed without merging**